.github/workflows/publish-hotfix-release.yml

```diff
@@ -0,0 +1,267 @@
+        required: true
+
+jobs:
+  # Gate: Check release version presence
+  validate-release:
+    name: Validate if hotfix can be released
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref_name, 'hotfix/')
+    steps:
+      - uses: mukunku/tag-exists-action@v1.6.0
+        id: check-tag
+        with:
+          tag: ${{ inputs.version }}
+
+      - if: steps.check-tag.outputs.exists == 'true'
+        name: Exit on tag existance
+        run: |
+          echo "Tag already exists! Please choose another tag."
+          exit 1
+
+  release-version:
+    name: Determine release version
+    needs: [validate-release]
+    runs-on: ubuntu-latest
+    outputs:
+      RELEASE_VERSION: ${{ steps.release-version.outputs.RELEASE_VERSION }}
+    steps:
+      - name: Output release version
+        id: release-version
+        run: |
+          echo "RELEASE_VERSION=${{ inputs.version }}" >> $GITHUB_OUTPUT
+
+  prepare-hotfix-release:
+    name: "Prepare hotfix release"
+    runs-on: ubuntu-latest
+    needs: [release-version]
+    permissions:
+      contents: write
+      packages: write
+      pages: write
+      pull-requests: write
+    if: needs.release-version.outputs.RELEASE_VERSION
+    steps:
+      - uses: actions/checkout@v4
+      - name: Initialize mandatory git config
+        run: |
+          git config user.name "eclipse-tractusx-bot"
+          git config user.email "tractusx-bot@eclipse.org"
+      - uses: ./.github/actions/setup-java
+      - name: Bump version in gradle.properties
+        run: |-
+          # replace the project's (default) version, could be overwritten later with the -Pversion=... flag
+          sed -i  's/version=.*/version=${{ needs.release-version.outputs.RELEASE_VERSION }}/g' gradle.properties
+        env:
+          GITHUB_PACKAGE_USERNAME: ${{ github.actor }}
+          GITHUB_PACKAGE_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+      - name: Bump version in /charts
+        uses: mikefarah/yq@v4.44.2
+        with:
+          cmd: |-
+            find charts -name Chart.yaml -maxdepth 3 | xargs -n1 yq -i '.appVersion = "${{ needs.release-version.outputs.RELEASE_VERSION }}" | .version = "${{ needs.release-version.outputs.RELEASE_VERSION }}"'
+      - name: Update Chart READMEs
+        uses: addnab/docker-run-action@v3
+        with:
+          image: jnorwood/helm-docs:v1.10.0
+          options: -v ${{ github.workspace }}/charts:/helm-docs
+          run: |
+            helm-docs --log-level debug
+      - name: Commit manifest files
+        id: make-commit
+        run: |
+          git add gradle.properties $(find charts -name Chart.yaml) $(find charts -name README.md)
+          git commit --message "Prepare release ${{ needs.release-version.outputs.RELEASE_VERSION }}"
+
+          echo "commit=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
+
+  # Release: Maven Artifacts
+  maven-release:
+    name: Publish extension's release version to maven repository
```

Comment:
Does it make sense to reuse the workflow `./.github/workflows/trigger-maven-publish.yaml` ?

Reply:
We don't have that on this branch :). ignore this :)